src/store/sagas.js

```diff
@@ -1,12 +1,12 @@
 // @flow
 
-import coreSagas from 'routes/sagas'
-import upgradeSagas from 'routes/Upgrade/sagas'
-import walletsSagas from 'routes/Wallets/sagas'
-import notFoundSagas from 'routes/NotFound/sagas'
-import favoritesSagas from 'routes/Favorites/sagas'
-import digitalAssetsSagas from 'routes/DigitalAssets/sagas'
-import settingsSagas from 'routes/Settings/sagas'
+import coreSagas from 'store/sagas/index'
+import { upgradeRootSaga as upgradeSagas } from 'store/sagas/upgrade'
+import walletsSagas from 'store/sagas/wallets'
+import * as notFoundSagas from 'store/sagas/notFound'
+import { favoritesRootSaga as favoritesSagas } from 'store/sagas/favorites'
+import * as digitalAssetsSagas from 'store/sagas/digitalAssets/core'
+import { settingsRootSaga as settingsSagas } from 'store/sagas/settings'
```

Comment:
I would like to left **clean** imports as is.
Combination of clean imports, `* as` and { fooRootSaga as fooSagas }` looks really bad.